src/lib.rs

```diff
@@ -718,10 +718,16 @@ fn format_code_block(code_snippet: &str, config: &Config) -> Option<String> {
     let mut result = String::with_capacity(snippet.len());
     let mut is_first = true;
 
+    // While formatting the code, ignore the config's newline style setting and always use "\n"
+    // instead of "\r\n" for the newline characters.
```

Comment:
Why is it OK to do this? It seems like this would insert the wrong line ending if the user wants Windows line endings.